client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
Oh yeah look at the login-form.hbs I actually render this as a partial. So it takes on the context of the template it's included in. 

Figured this would be very common and don't know if it needs to a component just include it partially.